sdk/keyvault/azure-security-keyvault/tests/test_examples_keys.py

```diff
@@ -148,9 +143,6 @@ def test_example_key_list_operations(self, vault_client, **kwargs):
         for key in keys:
             print(key.id)
             print(key.name)
-            print(key.created)
```

Comment:
What's the rationale for removing this?

Reply:
I didn't think there was much value in showing the created field for list operations.